weblab/templates/entities/entity_version.html

```diff
@@ -40,12 +41,18 @@ <h2 id="entityversionname">
             <a href="{{ entity|url_tag_version:version }}">Add tag</a>.
             <form method="POST" action="">
               Change visibility: {{ form.visibility }}
-              <img src="{% static 'img/info.png' %}" alt="help" title="{{ entity.VISIBILITY_HELP }}" />.
+              <img src="{% static 'img/info.png' %}" alt="help" title="{{ entity.VISIBILITY_HELP }}" />
               <span id="versionVisibilityAction"></span>
-            </form>
-          {% else %}
+                {% if perms.create_experiment %}
```

Comment:
I think this `if` block needs to go outside the `if permission` block, so it's after line 56 below. You can run experiments using an entity even if you can't make changes to that entity.

Reply:
Taken it out for now. Not sure where it does fit. I was going with the logic that by selecting combinations of model/protocols you were creating experiments and that if you did not have permission to do that there was no point in offering the option. I had thought we might need to filter the lists of offered combinations to ones that a user was allowed to 'create'  and 'run' - presumably only ones they already ran - if they dont have permission to run other peoples